python/GafferUI/_StyleSheet.py

```diff
@@ -894,11 +893,40 @@ def styleColor( key ) :
 		spacing: 5px;
 	}
 
-	QTreeView QHeaderView {
-		/* tuck header border inside the treeview border */
-		margin-top: -1px;
-		margin-left: -1px;
-		margin-right: -1px;
+	/* Avoid a double border with the tree view's border */
+
+	QTreeView QHeaderView::Section:horizontal {
```

Comment:
Section -> section. It does seem to work with the capitalisation, but I'm sure the next person to debug a stylesheet will be grateful for one less thing to consider.

Reply:
Oh good spot, thanks. Fixed in https://github.com/GafferHQ/gaffer/pull/3697/commits/48e1459d2c81e87a7f35a555f8f63522becb5be3